content/docs/sdk/flutter/introduction.md

```diff
@@ -20,8 +20,8 @@ If you haven't already, we recommend you head to [Get Started Flutter Developmen
 The following is covered in this codelab:
 
 - **AtClientManager** - an object with various services such as Client, NotificationService, and SyncService.
-- **AtClientPreference** - object used to configure preferences in your atPlatform application such as the namespace, hiveStoragePath, or maxDataSize.
-- **Onboarding** - process of activating an atSign and/or authenticating the atSign into its secondary server.
+- **AtClientPreference** - object used to configure preferences in your atPlatform application such as the namespace, hivestoragepath, or maxdatasize.
```

Comment:
`atsign` -> `atSign`
`hivestoragepath` -> `storage path for the hive database`
`maxdatasize` -> `maximum data size`

If they aren't going to remain as camel case arguments, I believe it's best we describe them fully.